src/OSPSuite.Infrastructure.Import/Core/DataFormat/DataFormatNonmem.cs

```diff
@@ -8,12 +8,10 @@ namespace OSPSuite.Infrastructure.Import.Core.DataFormat
 {
    public class DataFormatNonmem : AbstractColumnsDataFormat
    {
-      private const string _name = "Nonmem";
-      private const string _description = "https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/797";
-      public override string Name => _name;
-      public override string Description => _description;
+      public override string Name => "Nonmem";
+      public override string Description => "https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/797";
```

Comment:
@rwmcintosh Same question: why not to the docu?

Reply:
Rob doesn't know. This comes from before lol

Reply:
where are those descriptions used - probably showed as a tooltip somewhere?

Reply:
@Yuri05 i didn't investigate the property usage. I only noticed that there were these unnecessary private fields that I wanted to remove.